Libraries/LibC/bits/FILE.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <sys/cdefs.h>
+#include <sys/types.h>
+
+#define BUFSIZ 1024
+
+__BEGIN_DECLS
+
+struct __STDIO_FILE {
+    int fd;
+    int eof;
+    int error;
+    int mode;
+    pid_t popen_child;
+    char* buffer;
+    size_t buffer_size;
+    size_t buffer_index;
+    int have_ungotten;
+    char ungotten;
+    char default_buffer[BUFSIZ];
+};
+
+typedef struct __STDIO_FILE FILE;
+
+__END_DECLS
```

Comment:
`\n` here please :)

And also a copyright header in all the new files, I'm guessing?

Reply:
> And also a copyright header in all the new files, I'm guessing?

Is this a new thing???

Reply:
Yeah

Serenity's now old enough to how have her own copyright headers ;)

Reply:
The commit was https://github.com/SerenityOS/serenity/commit/94ca55cefdc61940c06bcf812060ff65f9be8bae

Reply:
Cheers I'll add em.